modules/help_editor/templates/menu_help_editor.tpl

```diff
@@ -1,54 +0,0 @@
-<script type="text/javascript" src="{$baseurl}/js/filterControl.js"></script>
```

Comment:
you can remove the entire file. just add `public $skipTemplate = true` to the PHP class